test/functional/ui/input_spec.lua

```diff
@@ -411,7 +411,7 @@ describe('skip do you really want to write on force write', function()
     os.remove('Xtest-overwrite-forced')
   end)
 
-  it("does not ask for confirmation when force write", function()
+  it("w! does not show \"file has been changed\" warning", function()
```

Comment:
just use single quotes. don't do this escaping stuff in descriptions.
```suggestion
  it(':w! does not show "file has been changed" warning', function()
```